src/main/java/model/AssessScore.java

```diff
@@ -39,7 +39,7 @@ public AssessScore(int key){
 
     int widerobj = 0;
     String widerobjcomment = null;
-    int widerobjweight = 0;
+    //int widerobjweight = 0;
```

Comment:
Farhan,

As we are taking full value of widerObjective, If we change the percentage in assessment.scala.html to 100%, We don't need to change any code in scala